client/src/reducers/teamReducer.js

```diff
@@ -21,6 +21,7 @@ const INITIAL_TEAM_STATE = {
   themeId: '',
   repoCreated: false,
   error: null,
+  retrievedAccessToken: false,
```

Comment:
I think this variable can straight up be removed because the only place it was being used was in DeployPage.js

Reply:
Added because we're referencing it in L53: https://github.com/Researchify/Researchify/blob/fd5babb72485a8c34041f8681e25af4e042eec1e/client/src/reducers/teamReducer.js#L53